redisraft.h

```diff
@@ -580,7 +583,20 @@ static int inline checkInRedisModuleCall(void) {
     return redis_raft_in_rm_call;
 }
 
+typedef struct JoinLinkState {
```

Comment:
Now that's it's totally generic, maybe this should be named something like `NodeExecuteCommandState` or something similar?

Reply:
that's less descriptive to me.  i.e. this is state that relates to joining/linking redis raft nodes/clusters to each other.    I'm unsure what nodeExecuteCommand actually means